mintpy/defaults/smallbaselineApp.cfg

```diff
@@ -75,12 +75,16 @@ mintpy.network.excludeIfgIndex = auto  #[1:5,25 / no], auto for no, list of ifg
 mintpy.network.referenceFile   = auto  #[date12_list.txt / ifgramStack.h5 / no], auto for no
 
 ## 2) Coherence-based network modification = (threshold + MST) by default
-## It calculates a average coherence for each interferogram using spatial coherence based on input mask (with AOI)
+## coherenceBased calculates an average coherence for each interferogram using spatial coherence based on input mask (with AOI)
+## Alternatively, areaBased calculates the proportional area of each interferogram that is above a spatial coherence threshold
+## This threshold is defined as the spatial coherence of the interferograms within the input mask
 ## Then it finds a minimum spanning tree (MST) network with inverse of average coherence as weight (keepMinSpanTree)
 ## Next it excludes interferograms if a) the average coherence < minCoherence AND b) not in the MST network.
 mintpy.network.coherenceBased  = auto  #[yes / no], auto for no, exclude interferograms with coherence < minCoherence
+mintpy.network.areaBased       = auto  #[yes / no], auto for no, exclude interferograms with valid area < minArea
 mintpy.network.keepMinSpanTree = auto  #[yes / no], auto for yes, keep interferograms in Min Span Tree network
-mintpy.network.minCoherence    = auto  #[0.0-1.0], auto for 0.7
+mintpy.network.minCoherence    = auto  #[0.0-1.0], auto for 0.7, use with coherenceBased
+mintpy.network.minArea         = auto  #[0.0-1.0], auto for 0.75, use with areaBased
```

Comment:
We should separate the options between coherence-based and area-based (effective coh ratio) methods. If I understood it right, `maskFile / aoiYX / aoiLALO` are shared among the two, `coherenceBased / minCoherence / keepMinSpanTree` are for coherence-based only, `areaBased / minArea` are for area-based only. Then, let's put them into 3 independent paragraphs:

```
2) Data-driven network modification
# some description
mintpy.network.maskFile
mintpy.network.aoiYX
mintpy.network.aoiLALO

2.1) Coherence-based network modification
# some description
mintpy.*

2.2) Effective coherence ratio
# some description
mintpy*
```

Reply:
I would also recommend using `minAreaRatio` instead of `minArea` to be clear it's the normalized value.